.prospector.yaml

```diff
@@ -0,0 +1,24 @@
+test-warnings: true
```

Comment:
This file is the config file for the prospector
I need to disable some of the messages since removing them causes errors in execution